### PR TITLE
user profile in employee page

### PR DESCRIPTION
**Pull request by akriti12negi**
#2113 

### Description
added a basic detail page in hr module which shows the basic details of a user  

### Checklist
 I have performed a self-review of my own code.

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -56,6 +56,17 @@
...
+        Route::resource('employees', 'Employees\EmployeeController')
+            ->only(['index', 'show'])
+            ->names([
+                'index' => 'employees',
+                'show' => 'employees.show',
+        ]);
+
+        Route::get('employee/{employee}/projects/', 'Employees\EmployeeController@showProjects')->name('employees.projects');
+
+        Route::get('employee-reports', 'Employees\ReportsController@index')->name('employees.reports');
```
Comment (GauravGusain98) [inline routes/web.php:69]: What these 2 routes are being used for?

Comment (GauravGusain98) [reply routes/web.php:69]: If these are not being used then please remove.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments

File: resources/views/hr/employees/menu.blade.php [hunk 2]
```diff
@@ -5,4 +5,7 @@
     <li class="nav-item">
         <a class="nav-item nav-link {{ Request::is('hr/employee-reports*') ? 'active' : '' }}" href="{{ route('employees.reports') }}"><i class="fa fa-pie-chart"></i>&nbsp;Reports</a>
     </li>
+    <li class="nav-item">
+        <a class="nav-item nav-link {{ Request::is('hr/employee-basic-details*') ? 'active' : '' }}" href="{{ route('employees.basic.details')}}"><i class="fa fa-details"></i>&nbsp;Basic Details</a>
```
Comment (srivastava-jyoti) [inline resources/views/hr/employees/menu.blade.php:9]:
```suggestion
        <a class="nav-item nav-link {{ Request::is('hr/employee-basic-details*') ? 'active' : ' ' }}" href="{{ route('employees.basic.details') }}"><i class="fa fa-details"></i>&nbsp;Basic Details</a>
```

File: resources/views/hr/employees/basic-details.blade.php [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            <div>
+                <div class="form-group">
+                    <label class="font-weight-bold" for="">Name:</label>
+                    <span>{{ $user->name }}</span>
+                </div>
+                <div class="form-group">
+                    <label class="font-weight-bold" for="">Email:</label>
+                    <span>{{ $user->email }}</span>
+                </div>
+                @includeWhen($user->profile, 'user::profile.subviews.show-user-profile-info')
```
Comment (srivastava-jyoti) [inline resources/views/hr/employees/basic-details.blade.php:22]: What is the use of `@includeWhen`?

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
@akriti12negi I am getting this error. Its looking like you are not passing the employee id to the route and it is required there. Please fix this
image

**Review by GauravGusain98 — APPROVED** [review]
Looks fine

**Pull request merged**